mindmeld/components/entity_resolver.py

```diff
@@ -1381,28 +1017,12 @@ def _predict(self, nbest_entities, top_n):
             (list): The resolved values for the provided entity.
         """
 
-        synonyms, synonyms_encodings = self._synonyms, self._synonyms_embs
-
         # encode input entity
         # TODO: Use all provided entities (i.e all nbest_entities) like elastic search
         top_entity = nbest_entities[0]  # top_entity
-        existing_index = synonyms.get(top_entity.text, None)
-        if existing_index:
-            top_entity_emb = synonyms_encodings[existing_index]
-        else:
-            top_entity_emb = (
-                self.encode(
-                    top_entity.text,
-                    concat_last_n_layers=self._runtime_configs["concat_last_n_layers"],
-                    normalize_token_embs=self._runtime_configs["normalize_token_embs"],
-                )
-            )
-        top_entity_emb = top_entity_emb.reshape(1, -1)
 
         try:
-            sorted_items = self._compute_cosine_similarity(
-                synonyms, synonyms_encodings, top_entity_emb, top_n
-            )
+            sorted_items = self._embedder_model.find_similarity(top_entity.text, _no_sort=True)
```

Comment:
Is it better to add a double negative like `_no_sort` vs `sort`?

Reply:
So the use-case is to avoid sorting; sorting happens by default unless _no_sort is called. I felt it makes sense to call it _no_sort for that reason only.